tests/spread/multipass/error-handling/task.yaml

```
@@ -25,5 +25,6 @@ execute: |
   # Ensure that snapcraft uses build VMs
   unset SNAPCRAFT_BUILD_ENVIRONMENT
 
-  snapcraft
-  [ $? -eq 2 ]
+  err=0
```

how about using the same pattern as everywhere else

```suggestion
  if ! snapcraft; then
    echo "Expected <fill the blank>"
    exit 1
  fi
```

If we care about the code, then use your construct, but add some messaging please

---

The expected result is, in this case, 2. Result will be 1 if the error is not properly handled, so I'll keep the construct and add an error message.